spa/src/pages/mall/creator/catalog.vue

```diff
@@ -108,6 +130,13 @@ export default Vue.extend({
         this.objects = response.data.object.objects;
         this.total = response.data.object.total[0].count;
         this.showSuccess = true;
+        let pages = Math.round(this.total/this.limit);
```

Comment:
Would we want this to consistently round up/down or is it ok to round to the nearest integer?  Just wondering if Math.ceil or Math.floor would give more of a consistent result.

Reply:
Made requested changes.